core/src/main/java/hudson/model/Queue.java

```diff
@@ -1383,12 +1387,13 @@ public void maintain() {
                 // of isBuildBlocked(p) will become a bottleneck before updateSnapshot() will. Additionally
                 // since the snapshot itself only ever has at most one reference originating outside of the stack
                 // it should remain in the eden space and thus be cheap to GC.
-                // See https://issues.jenkins-ci.org/browse/JENKINS-27708?focusedCommentId=225819&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-225819
+                // See https://issues.jenkins-ci.org/browse/JENKINS-27708?focusedCommentId=225819&page=com.atlassian
+                // .jira.plugin.system.issuetabpanels:comment-tabpanel#comment-225819
```

Comment:
`https://issues.jenkins-ci.org/browse/JENKINS-27708?focusedCommentId=225819` should just work